bld/namelist_files/namelist_definition_mpassi.xml

```diff
@@ -2565,6 +2565,18 @@ Default: Defined in namelist_defaults.xml
 	category="prescribed_ice" group="prescribed_ice">
 Prescribed ice stream parameter for e3sm sea ice driver.
 
```

Comment:
I don't think this entry for `config_prescribed_ice_stream_fldfilename` is closed appropriately, and it still needs the `Valid values:` and `Default:` keys.

Reply:
Heh, fixed this after I made the previous commit. It's there now.

Reply:
Agreed, thanks!